cmd/helm/root.go

```diff
@@ -42,7 +42,11 @@ Common actions from this point include:
 - helm list:      list releases of charts
 
 Environment:
-  $HELM_HOME          set an alternative location for Helm files. By default, these are stored in ~/.helm
+  $HELM_HOME          set an alternative location for Helm files. By default, these are stored in 
+					  "$XDG_CONFIG_DIR/helm"" on *nix, "%APPDATA%\helm" on Windows and 
```

Comment:
can we list out what $XDG_CONFIG_DIR is on Linux? Something like

```
$HELM_HOME          set an alternative location for Helm files. By default, these are stored in 
					  "$XDG_CONFIG_DIR/helm" (typically $HOME/.config/helm) on *nix, "%APPDATA%\helm" on Windows and
```

Also there's an extra double quote after `$XDG_CONFIG_DIR/helm`. Would you mind removing that please?

Reply:
Fixed :smile: